Source/fspathtools.c

```diff
@@ -9,6 +9,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#include <linux/limits.h>
```

Comment:
This breaks Cygwin compilation. Also this file already had this file included at https://github.com/hfiref0x/LightFTP/blob/36aae4bc1b10bafe34e4d034d290161e9845345e/Source/fspathtools.c#L8

Reply:
In fact, the first failure is because PATH_MAX is not defined. If compatibility is considered, maybe adding a PATH_MAX is a suitable solution?

Reply:
I've replied below.